styles/pronouns.page/catppuccin.user.css

```diff
@@ -212,6 +212,11 @@
       background-color: @accent-color !important;
     }
 
+    button:disabled {
+       background-color: darken(@accent-color, 5%);
+       color: @crust;
+    }
```

Comment:
This actually looks wrong. Look at how buttons and disabled buttons/colors are themed generally in the lines above.

Reply:
what if we use ``.btn`` or ``.btn-primary`` instead of ``button``?

Reply:
Right can you move and adjust this there? The selector they use to theme disabled buttons should be the same we use in most cases.

Reply:
@uncenter uh sure

Reply:
@uncenter wait what if we can just use variables instead of manually theming the disabled buttons?

Reply:
"uh sure ig" after me telling you yes, you should "use .btn or .btn-primary instead of button for this" (your words), is wild.

Reply:
> "uh sure ig" after me telling you yes, you should "use .btn or .btn-primary instead of button for this" (your words), is wild lmao.

its because i didn't understand some parts of your comment lol

Reply:
also i dug deeper into the code of this userstyle and found out you only need ``--bs-btn-disabled-bg`` to theme the disabled buttons, can't we just use that instead of manually theming the disabled buttons?

Reply:
Yep that's what I meant by adjusting. In the future you should *always* check for things like that - don't just theme the first thing that "fixes" it, go back up the element tree and find the one with the changes/miscoloration and see what caused it.

Reply:
yeah i'll do that in the near future, im also going to follow some feedback from bell too

Reply:
I hope so!

Reply:
> I hope so!

im also gonna try not reverting back after following the feedback, which is probably why i got banned from the discord